src/main/groovy/com/ullink/gradle/sonar/SonarScannerMsbuildPlugin.groovy

```diff
@@ -35,15 +45,15 @@ class SonarScannerMsbuildPlugin implements Plugin<Project> {
         project.gradle.taskGraph.whenReady { graph ->
             if (graph.hasTask("${project.path}:${SONAR_SCANNER_TASK_NAME}") || graph.hasTask(":${SONAR_SCANNER_TASK_NAME}")) {
                 project.logger.info("${SONAR_SCANNER_TASK_NAME} task was detected. Initializing sonar-scanner-msbuild...")
-                initSonarScanner(project, scannerExtension)
+                initSonarScanner(project)
             }
         }
     }
 
-    private void initSonarScanner(Project project, SonarScannerMsbuildExtension extension) {
+    private void initSonarScanner(Project project) {
         downloadSonarScanner(project)
         project.exec {
-            commandLine = [getSonarScannerFile(project), 'begin'] + buildArgs(project, extension.properties.properties)
+            commandLine = [getSonarScannerFile(project), 'begin'] + buildArgs(computeSonarProperties(project))
```

Comment:
Do you know if sonarqube supports the help command? /?  or /h

Reply:
In that command sonar-scanner-msbuild is ran which has the /? and /h arguments:
```
C:\sonar-scanner-msbuild\4.6.0.1930>SonarScanner.MSBuild.exe /h
SonarScanner for MSBuild 4.6
Using the .NET Framework version of the Scanner for MSBuild

Usage:

  SonarScanner.MSBuild.exe [begin|end] /key:project_key [/name:project_name] [/version:project_version] [/d:sonar.key=value] [/s:settings_file]

    When executing the begin phase, at least the project key must be defined.
    Other properties can dynamically be defined with '/d:'. For example, '/d:sonar.verbose=true'.
    A settings file can be used to define properties. If no settings file path is given, the file SonarQube.Analysis.xml in the installation directory will be used.
    Only the token should be passed during the end phase, if it was used during the begin phase.
```